### ISSUE TITLE
Move the sliders / reset the filter with code

### ISSUE THREAD

**Issue by olitomas** [JackDanielsAndCode/react-native-multi-slider#14]
It would be great if I could set the slider value with coding (not my finger). On the same note it would be good to have a "Reset filters" function that resets the filters to their original state.

**Comment by l-urence** [comment]
https://github.com/JackDanielsAndCode/react-native-multi-slider/pull/18 should enable you to set the slider values programmatically. Right now without an animation but maybe there will be a future implementation.

**Comment by gudmundurmar** [comment]
@l-urence 
This merge doesn't seem to have been released in a new npm version...
I couldn't see it in 0.2.1 or 0.2.2 at least.

**Comment by l-urence** [comment]
@jrans I think the npm needs an update/new release. @gudmundurmar In the mean time you can install the package from github

```bash
$ npm install --save JackDanielsAndCode/react-native-multi-slider
```

**Comment by gudmundurmar** [comment]
@l-urence  Yes, that's what I ended up doing :)
It's just not a nice solutions in case there are any breaking changes introduced to the master.
When that's done I think we can close this issue @olitomas

**Comment by jrans** [comment]
@l-urence @gudmundurmar npm version updated, sorry for teh delay and thanks for your patience!

**Comment by gudmundurmar** [comment]
@jrans Great :)

**Comment by gudmundurmar** [comment]
I have a question regarding this issue. 
If you create an initial arrayValues/optionsArray that has [1,2,3], but then receive new values through props like 4. 
I get ```Invalid value, array does not contain: 4```
Doesn't the arrayValue/optionsArray need to update as soon as the new props are received?

**Comment by l-urence** [comment]
According to the README:
> Note these must be possible values from your set up.

Maybe you can can update the `optionsArray` prop with your new value(s)? As far as I can see the slider will then use your new `optionsArray` (https://github.com/JackDanielsAndCode/react-native-multi-slider/blob/master/Slider.js#L101).

**Comment by gudmundurmar** [comment]
Had a better look at this and found the problem...
Since the values are changing then the min, max can also change.
To fix this the  set(values) from https://github.com/JackDanielsAndCode/react-native-multi-slider/blob/master/Slider.js#L101
should also receive set(values, min, max). Otherwise making a new optionsArray will just use the old this.props.min, this.props.max and then the optionsArray will be wrong. 
I made pull request that addresses this problem. https://github.com/JackDanielsAndCode/react-native-multi-slider/pull/22

**Comment by l-urence** [comment]
Fixed with #22.
